README.md

```diff
@@ -74,19 +72,9 @@ give rewards for reported issues, according to impact and severity.
 * [@jcarpanelli](https://github.com/jcarpanelli)
 * [@ylv-io](https://github.com/ylv-io)
 
-## Development setup
-
-Run `npm install` in the root of the project, followed by `npx lerna bootstrap` to have lerna install all dependencies in all packages and symlink them as needed.
```

Comment:
I'd leave a section on development setup that just indicates that the project is bootstrapped by running `yarn` in the root of the project, and refer to the CONTRIBUTING guide for more info.

Reply:
I have added a line to the **Contribute** section.